dockers/docker-lldp-sv2/Dockerfile.j2

```diff
@@ -10,7 +10,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update
 
 # Install dependencies
-RUN apt-get install -y python-pip libbsd0 libevent-2.0-5 libjansson4 libwrap0 libxml2 libpci3 libperl5.20 libpython2.7
+RUN apt-get install -y adduser init-system-helpers libbsd0 libevent-2.0-5 libjansson4 libsensors4 libwrap0 libxml2 libpci3 python-pip lsb-base libpython2.7
```

Comment:
Can you please explain why we now need to install adduser, init-system-helpers, libsensors4 and lsb-base?

Reply:
Hey Joe,
I just followed the requirement to build lldpd as per below docs.
**Link:** [https://packages.debian.org/stretch/lldpd](url)

so, added those which were not there in the dockerfile.

Reply:
Thank you for paying attention to these dependencies. However, I'm not sure if it's necessary to install them explicitly here. Did you try building an image without adding them? The `RUN dpkg_apt() ...` lines below should install all built packages _as well as_ their dependencies. I checked the lldpd dependencies for Debian Jessie and most of the same dependencies are listed, however we never explicitly installed them, yet everything built and worked just fine.

The idea behind adding the `RUN dpkg_apt() ...` lines is that we will not need to maintain the list of dependencies going forward. Actually, I wonder if anything is necessary in that list besides `python-pip` simply for installing Python Wheels. Could you please see how minimal you can make that list?

Reply:
Hey Joe, 
I just removed all the extra packages I installed for lldpd in the DockerFile. You are correct . We don't need this explicitly.  I will remove this. Thanks for your explanation!

Reply:
You're welcome. Are you able to remove any of the pre-existing dependent packages, also?